lib/ex_machina/ecto.ex

```diff
@@ -133,4 +134,10 @@ defmodule ExMachina.Ecto do
       {name, _type} -> Map.delete(map, name)
     end
   end
+
+  defp drop_nil_values(map) do
+    map
+    |> Enum.reject(fn({_, v}) -> v == nil end)
```

Comment:
This looks great. Could you change `v` to `value`. We are trying to avoid single letter values so that it's clear to future contributors what's going on